libs/ui/src/accessible_controllers/accessible_controller_sandbox.test.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  advanceTimers,
+  fakeUseAudioControls,
+  mockOf,
+} from '@votingworks/test-utils';
+import React from 'react';
+import { assert } from '@votingworks/basics';
+import userEvent from '@testing-library/user-event';
+import { UiString } from '../ui_strings/ui_string';
+import {
+  AccessibleControllerHelpStrings,
+  AccessibleControllerIllustrationProps,
+  AccessibleControllerSandbox,
+} from './accessible_controller_sandbox';
+import { act, screen } from '../../test/react_testing_library';
+import { Keybinding } from '../keybindings';
+import { newTestContext } from '../../test/test_context';
+
+const mockAudioControls = fakeUseAudioControls();
+
+jest.mock(
+  '../ui_strings/ui_string',
+  (): typeof import('../ui_strings/ui_string') => ({
+    ...jest.requireActual('../ui_strings/ui_string'),
+    UiString: jest.fn(),
+  })
+);
+
+jest.mock(
+  '../hooks/use_audio_controls',
+  (): typeof import('../hooks/use_audio_controls') => ({
+    ...jest.requireActual('../hooks/use_audio_controls'),
+    useAudioControls: () => mockAudioControls,
+  })
+);
+
+function newRenderer() {
+  let lastReadElement: HTMLElement;
+
+  const mockOnClick = jest.fn().mockImplementation((event: MouseEvent) => {
+    assert(event.target instanceof HTMLElement);
+    lastReadElement = event.target;
+  });
+
+  const { render } = newTestContext();
+
+  function renderWithMockScreenReader(ui: React.ReactNode) {
+    return render(<div onClickCapture={mockOnClick}>{ui}</div>);
+  }
+
+  return {
+    expectLastMockScreenReaderContent: (textContent: string) =>
+      expect(lastReadElement).toHaveTextContent(textContent),
+    renderWithMockScreenReader,
+  };
+}
+
+function simulateKeyPress(key: string) {
+  userEvent.keyboard(key.length === 1 ? key : `[${key}]`);
```

Comment:
nit: factor out for DRYness? Similar implementation to https://github.com/votingworks/vxsuite/pull/4740/files#diff-4d3153775fb1fec32f94bd89603d0bcf0828ed2d79f72cefcc61d807dc23b5d6R26

Reply:
+1 - will do